tests/test_copy_to_aws.py

```
@@ -15,15 +126,83 @@ def test_collect_upload_tasks(tmp_path):
+        def __init__(self, renamed_path):
+            self.paths = types.SimpleNamespace(renamed=renamed_path)
+
+        def get(self, key, default=None):
+            return default
+
+    cfg = DummyCfg(tmp_path)
+    missing = tmp_path / "missing_manifest.csv"
+
+    try:
+        collect_upload_tasks_from_manifest(missing, [".jpg", ".jpeg"], cfg, secured_mode=False)
+        assert False, "Expected FileNotFoundError"
```

Ruff (0.15.14)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)